modules/rosa-policy-incident.adoc

```diff
@@ -40,8 +40,19 @@ The following activities can trigger notifications:
 - Critical vulnerabilities and resolution
 - Upgrade scheduling
 
-[id="rosa-policy-backup-recovery_{context}"]
-== Backup and recovery
+[id="rosa-policy-backup-recovery-sts_{context}"]
+== Backup and recovery for ROSA clusters with STS
+
+There is no backup method available for ROSA clusters with STS at this time.
+
+[WARNING]
+====
+Etcd backup does not apply to STS clusters.
```

Comment:
Do we still need this? Since this note would be covered by the newly added statement.